Vagrantfile

```diff
@@ -75,8 +79,12 @@ Vagrant.configure("2") do |config|
         config.vm.network "forwarded_port", guest: 2375, host: ($expose_docker_tcp + i - 1), auto_correct: true
       end
 
-      config.vm.provider :vmware_fusion do |vb|
-        vb.gui = $vb_gui
+      ["vmware_fusion", "vmware_workstation"].each do |vmware|
+        config.vm.provider vmware do |v|
+          v.gui = $vb_gui
+          v.vmx["memsize"] = $vb_memory
```

Comment:
FWIW, this is how I've done this before: https://github.com/c4milo/osx-builder/blob/master/vms/vm.go#L166